packages/schemas/tables/sign_in_experiences.sql

```diff
@@ -9,6 +10,7 @@ create table sign_in_experiences (
   language_info jsonb /* @use LanguageInfo */ not null,
   terms_of_use_url varchar(2048),
   privacy_policy_url varchar(2048),
+  agree_to_terms_policy agree_to_terms_policy not null default 'AutoAgree',
```

Comment:
by default we are not auto-agree?

Reply:
did you consider social sign-in? since some social sign-in already shows terms and policies

Reply:
aligned offline